scripts/nuget.ps1

```diff
@@ -38,9 +38,7 @@ function BuildEmbeddedNuget ($projectDir, $outDir, $serverSrcDir, $studioZipPath
     $EMBEDDED_OUT_DIR = [io.path]::combine($outDir, "RavenDB.Embedded")
     $EMBEDDED_BUILD_OUT_DIR = [io.path]::combine($EMBEDDED_OUT_DIR, "build")
     $EMBEDDED_SERVER_OUT_DIR = [io.path]::combine($EMBEDDED_OUT_DIR, "contentFiles", "any", "any")
-    
-    $LIB_ZSTD_DIR = [io.path]::combine($projectDir, "libs", "libzstd", "*")
```

Comment:
Don't we have to do the same for Libsodium?

Reply:
No, libsodium is not used by Client